courseware/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Courseware
+nav_order: 5
+has_children: true
+---
+
+# Courseware
+
+[Data Science with Real Data](/courseware/intro_data_science.html)
```

Comment:
how about if the link here reads: "Intro to Data Science"

Reply:
Agreed, implemented in new commit!